templates/Pages/about.php

```diff
@@ -41,6 +41,25 @@
         <li><strong>IO500 List</strong>: the subset from the full list with only the best submission per storage system</li>
         <li><strong>10 Node Challenge List</strong>: the subset from the full list with only the best submission per storage system ran on exactly ten nodes</li>
     </ul>
+    
+    <p>
+        Please note that the Ranked Lists will effectively <i>hide</i> some submissions due to the fact that they only show the best submission per storage systems.  Therefore, 
```

Comment:
I would say "can" or "may" hide.

Also, I would write "... they only show the submission with the highest aggregate score per storage system", since "best" is subjective and may not be clear to the reader what the actual criterion is.